specification/network/resource-manager/Microsoft.Network/stable/2022-05-01/customIpPrefix.json

```diff
@@ -390,16 +394,40 @@
             "Provisioning",
             "Provisioned",
             "Commissioning",
+            "CommissionedNoInternetAdvertise"
```

Comment:
Missing ',' at the end.
Please fix this otherwise tools are unable to parse the Swagger for the check.